modulus/models/graphcast/graph_cast_net.py

```diff
@@ -129,9 +129,19 @@ def __init__(
         use_cugraphops_decoder: bool = False,
         do_concat_trick: bool = False,
         recompute_activation: bool = False,
+        partition_size: int = 1,
+        partition_group_name: Optional[str] = None,
+        expect_partitioned_input: bool = False,
```

Comment:
When this is `True`, how would one account for that in the dataloader? Is there a mechanism for that? If not, we should think about how we can implement some such mechanisms in a future PR.

Reply:
Unfortunately, the dataloader side hasn't really been thought about. The main use-case for this would e.g. be multi-step roll-out, i.e. when the output is partitioned, the next-steps can `expect_partitioned_input` which for now would require re-setting this flag or doing a manual partitioning before the rollout

Reply:
Okay, that makes sense. Just opened https://github.com/NVIDIA/modulus/issues/151 to track this dataloader issue.

Reply:
Thanks, definitely makes sense 👍🏽